### ISSUE TITLE
Right-click menu for file search

### ISSUE THREAD

**Issue by fadilf** [appetizermonster/hain#162]
I often find myself searching for whatever I search for in Hain in the Windows search bar, even though I wish I could completely replace it with Hain. The right-click menu is extremely useful for things like opening file location, opening in programs, etc. This feature would be very useful.

**Comment by appetizermonster** [comment]
Nice idea, I'll add this into backlog

**Comment by blainesch** [comment]
Isn't using the mouse the opposite of what a launcher application is helping with? Alfred has commands like "open file.js" or "find file.js" to either open the file or open the file in finder.

**Comment by fadilf** [comment]
Good point. What if Hain could somehow simulate a right-click on the highlighted file on command?

**Comment by blainesch** [comment]
Alfred gives you a few prefixes like "open index.js" would open it in the default program and "find index.js" will open it in the finder. You could add other things for "delete", "move" or "rename"
